synapse/lib/slabseqn.py

```diff
@@ -40,7 +40,7 @@ def add(self, item, indx=None):
             return indx
 
         indx = self.indx
-        self.slab.put(s_common.int64en(indx), s_msgpack.en(item), db=self.db)
+        self.slab.put(s_common.int64en(indx), s_msgpack.en(item), append=True, db=self.db)
```

Comment:
Is there any circumstance under which this could *ever* be incorrect?  LMDB will literally create corrupt data structures if we're ever wrong about this.

Reply:
Nope. I've previously tested it out (and corrected the py-lmdb documentation). It simply returns failure if it is not the last key.

Reply:
awesome, is that something that the py-lmdb code would `raise` about or would we be expected to check a return value somewhere?

Reply:
Added asserts.